src/sample/courseRegController.java

```diff
@@ -345,48 +395,11 @@ public void updatebothlist() {
     displaycourselist();
   }
 
-  // When a Course in the Course List is Clicked, this function will be executed.
-  public void courselistdetails() {
-    if (courseList.getSelectionModel().getSelectedItem() != null) {
-
-      // Display the Course Details
-      courseTitle.setText(courseList.getSelectionModel().getSelectedItem());
-      try {
-        List<String> lines = Files
-            .lines(Paths.get("data/Course Details/" + courseList.getSelectionModel().getSelectedItem() + ".txt"))
-            .collect(Collectors.toList());
-        time.setText(lines.get(0));
-        desc.setText(lines.get(2));
-      } catch (IOException e) {
-        System.out.println("An error occurred.");
-        e.printStackTrace();
-      }
-    }
-  }
-
-  // When a Course in the Course Registered is Clicked, this function will be
-  // executed.
-  public void courseregistereddetails() {
-    if (registeredCourse.getSelectionModel().getSelectedItem() != null) {
-
-      // Display the Course Details
-      courseTitle.setText(registeredCourse.getSelectionModel().getSelectedItem());
-      try {
-        List<String> lines = Files
-            .lines(Paths.get("data/Course Details/" + registeredCourse.getSelectionModel().getSelectedItem() + ".txt"))
-            .collect(Collectors.toList());
-        time.setText(lines.get(0));
-        desc.setText(lines.get(2));
-      } catch (IOException e) {
-        System.out.println("An error occurred.");
-        e.printStackTrace();
-      }
-    }
-  }
-
   @Override
   public void initialize(URL location, ResourceBundle resources) {
     displaycourselist();
     displaycourseregistered();
+    addCourseButton.setDisable(true);
+    removeCourseButton.setDisable(true);
```

Comment:
Why disable the buttons?

Reply:
Here I put `.setDisable(true);` but then if the user selected a course again, then it will become `.setDisable(false);`
You can see `.setDisable(false);` in the `details` function. I tried le it works as expected.